images/openvswitch/Dockerfile

```diff
@@ -4,18 +4,13 @@
 # The standard name for this image is openshift/openvswitch
 #
 
-FROM centos:centos7
+FROM openshift/origin-base
```

Comment:
@sdodson I just tried to re-use the origin-base as it is just centos with couple extra packages... I can revert this and do manual verification.